architecture/core_concepts/routes.adoc

```diff
@@ -817,11 +818,32 @@ resolution order (oldest route wins).
 ====
 
 
-[[route-specific-timeouts]]
-== Route-specific Timeouts
+[[route-specific-annotations]]
+== Route-specific Annotations
 
-By default, a server-side timeout is set across all routes. It is possible to
-change that timeout using an annotation in the route configuration.
+By default, or using an environment variable a number of configuration options
```

Comment:
Maybe reword this - sounds a bit odd `by default or using` - something like `Using environment variables as defined in <add-link-to-section>, a router can set the default options for all the routes it exposes. An individual route can choose to override some of these defaults by providing specific configuration via/in (I like via but its normally not used in docs) its annotations`.